play.pokemonshowdown.com/src/panel-battle.tsx

```diff
@@ -1012,7 +1015,10 @@ class BattlePanel extends PSRoomPanel<BattleRoom> {
 				</p>
 			) : (
 				<p>
-					<button class="button" data-cmd="/switchsides"><i class="fa fa-random" aria-hidden></i> Switch viewpoint</button>
+					<button class="button" data-cmd="/switchsides"><i class="fa fa-random" aria-hidden></i> Switch viewpoint</button> {}
+					<button class="button" data-href="gototurnpopup">
```

Comment:
Hm. I think this would work better with a `PS.prompt` than with a custom popup. Your custom popup looks like it doesn't autofocus, either, and it's missing some styling.

Reply:
Plausibly this should go to `/ffto`, and then `/ffto` without a turn would open the prompt.